src/main/java/io/reactivex/flowables/ConnectableFlowable.java

```diff
@@ -78,19 +78,45 @@ public final Disposable connect() {
     }
 
     /**
-     * Returns a Flowable that automatically connects to this ConnectableFlowable
+     * Returns a Flowable that automatically connects (at most once) to this ConnectableFlowable
      * when the first Subscriber subscribes.
+     * <p>
+     * <img width="640" height="392" src="https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/autoConnect.f.png" alt="">
+     * <p>
+     * The connection happens after the first subscription and happens at most once
+     * during the lifetime of the returned Flowable. If this ConnectableFlowable
+     * terminates, the connection is never renewed, no matter how Subscribers come
+     * and go. Use {@link #refCount()} to renew a connection or dispose an active
+     * connection when all {@code Subscriber}s have cancelled their {@code Subscription}s.
+     * <p>
+     * This overload does not allow disconnecting the connection established via
+     * {@link #connect(Consumer)}. Use the {@link #autoConnect(int, Consumer)} overload
+     * to gain access to the {@code Disposable} representing the only connection.
      *
      * @return a Flowable that automatically connects to this ConnectableFlowable
      *         when the first Subscriber subscribes
+     * @see #refCount()
+     * @see #autoConnect(int, Consumer)
      */
     @NonNull
     public Flowable<T> autoConnect() {
         return autoConnect(1);
     }
     /**
-     * Returns a Flowable that automatically connects to this ConnectableFlowable
+     * Returns a Flowable that automatically connects (at most once) to this ConnectableFlowable
      * when the specified number of Subscribers subscribe to it.
+     * <p>
+     * <img width="640" height="392" src="https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/autoConnect.f.png" alt="">
+     * <p>
+     * The connection happens after the given number of subscriptions and happens at most once
+     * during the lifetime of the returned Flowable. If this ConnectableFlowable
+     * terminates, the connection is never renewed, no matter how Subscribers come
+     * and go. Use {@link #refCount()} to renew a connection or dispose an active
+     * connection when all {@code Subscriber}s have cancelled their {@code Subscription}s.
+     * <p>
+     * This overload does not allow disconnecting the connection established via
+     * {@link #connect(Consumer)}. Use the {@link #autoConnect(int, Consumer)} overload
+     * to gain access to the {@code Disposable} representing the only connection.
```

Comment:
nit: add `@see refCount() and another overload`?

Reply:
Same below + in `ConnectableObservable`

Reply:
I omitted `refCount` here because there is no `refCount(int)` yet and that would confuse readers.